lib/PrincipleStudios.OpenApi.CSharp/CSharpSchemaSourceResolver.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using HandlebarsDotNet;
 using Microsoft.OpenApi.Models;
+using PrincipleStudios.OpenApi.CSharp.templates;
```

Comment:
Is `templates` lowercased on purpose?

Reply:
Legacy reasons here, unfortunately; I'd like to correct a number of these in the future. (Definitely not for this PR.)